canonical-web-and-design/vanilla-framework
# Accordion accessibility update

## Done

- Deprecated previous accordion default pattern, and `.p-accordion__tab--with-title` pattern.
- Added a more accessible pattern that replaces both previous patterns

Fixes #3318 

## QA

- Visit [default accordion](https://vanilla-framework-3622.demos.haus/docs/examples/patterns/accordion/default)
- Check that each accordion tab functions as expected, and looks good
- Visit [headings accordion](https://vanilla-framework-3622.demos.haus/docs/examples/patterns/accordion/headings)
    - Check that there are no visual changes compared to [live](https://vanillaframework.io/docs/examples/patterns/accordion/headings)
- Verify deprecated patterns still work:
  - Visit [deprecated default accordion](https://vanilla-framework-3622.demos.haus/docs/examples/patterns/accordion/default-deprecated)
    - Check that there are no visual changes compared to [live](https://vanillaframework.io/docs/examples/patterns/accordion/default)
- Review updated documentation:
  - [Component status](https://vanilla-framework-3622.demos.haus/docs/component-status)
  - [Accordion](https://vanilla-framework-3622.demos.haus/docs/patterns/accordion)
